neofetch

```diff
@@ -13702,6 +13702,9 @@ main() {
     get_distro
     get_bold
     get_distro_ascii
+
+    # check if the output is a interactive terminal
+    [[ -t 1 ]] || stdout=on
```

Comment:
This would be a nice default, but if you write it this way, you won't be able to turn `stdout` off in non-tty mode. (e.g. when you actually want colors written to a file output)